docs/en_US/Compression/QuantizationSpeedup.rst

```diff
@@ -0,0 +1,141 @@
+Speed up Mixed Precision Quantization Model
+===========================================
+
+*This feature is in Beta version.*
+
+Introduction
+------------
+
+Deep learning network has been computationly intensive and memory intensive 
+which increase the difficulty of deploying deep neural network model. Quantization is a 
+fundamental technology which is widely used to reduce memory footprint and speed up inference 
+process. Many frameworks begin to support quantization, but few of them support mixed precision 
+quantiation. Frameworks like haq only support simulated mixed precision quantization which will 
```

Comment:
add reference for haq, otherwise it is not easy to understand this word...

Reply:
Have added reference.